data/pc/1.17/protocol.json

```diff
@@ -3085,6 +3085,13 @@
             {
               "name": "forced",
               "type": "bool"
+            },
+            {
+              "name": "promptMessage",
+              "type": [
```

Comment:
that's wrong, option already contains the bool

Reply:
I don't understand what you mean, I was trying to say this is an optional string arg, is that not what that means?

Reply:
yes but you need to remove the bool field

Reply:
option means bool + something
so you need to remove the bool above or there is 2 bool

Reply:
by bool field, do you mean this one? https://github.com/PrismarineJS/minecraft-data/blob/b3b1b88b4af060af1157478616ca24a3e5dc638b/data/pc/1.17/protocol.json#L3086-L3087 if so, that should still be there, if not, I'm still not understanding you